core/deployment/src/test/java/io/quarkus/deployment/pkg/steps/NativeImageBuildContainerRunnerTest.java

```diff
@@ -14,6 +15,9 @@ class NativeImageBuildContainerRunnerTest {
 
     @Test
     void testBuilderImageBeingPickedUp() {
+        // This will default to false in the maven build and true in the IDE, so this will still run if invoked explicitly
+        Assumptions.assumeFalse(Boolean.parseBoolean(System.getProperty("avoid-containers")),
+                "Skipping native image build container tests because -Dstart-containers is not set.");
```

Comment:
Can't we replace this change with 

```
@EnabledIfSystemProperty(named="avoid-containers", matches="true")
```

?

Reply:
Yes, that would be cleaner and more consistent with the rest of the codebase, thanks! It would have to be `@DisabledIfSystemProperty(named="avoid-containers", matches="true")` because of the inversion I did. 
(... which is slightly less consistent with the rest of the codebase, and more complicated, but it felt more desirable.)

Reply:
Good point!